### PR TITLE
Fall back to .lan address if .local doesn't work

### LINKED ISSUE #63: Deploy should fall back to .lan
If the mdns .local lookup fails, the deploy should try .lan.

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #63

### CODE REVIEW

**Comment by PeterJohnson** [comment]
@frcjenkins test this please

File: edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml [hunk 1]
```diff
@@ -31,27 +31,38 @@
...
 		<var name="target" unset="true"/>
-		<echo> roboRIO not found via mDNS, falling back to static USB</echo>
-		<property name="target" value="172.22.11.2"/>
+		<echo> roboRIO not found via mDNS .local, falling back to .lan</echo>
+	  	<property name="target" value="roboRIO-${team-number}-FRC.lan"/>
 		<if>
 			<isreachable host="${target}" timeout="5"/>
 		  <then>
-			<echo>roboRIO found via static USB</echo>
+			<echo>roboRIO found via mDNS .lan</echo>
```
Comment (Kevin-OConnor) [inline edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml]: The .lan lookup is DNS.

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml [hunk 2]
```diff
@@ -31,27 +31,38 @@
...
 		<var name="target" unset="true"/>
-		<echo> roboRIO not found via mDNS, falling back to static USB</echo>
-		<property name="target" value="172.22.11.2"/>
+		<echo> roboRIO not found via mDNS .local, falling back to .lan</echo>
+	  	<property name="target" value="roboRIO-${team-number}-FRC.lan"/>
 		<if>
 			<isreachable host="${target}" timeout="5"/>
 		  <then>
-			<echo>roboRIO found via static USB</echo>
+			<echo>roboRIO found via mDNS .lan</echo>
```
Comment (Kevin-OConnor) [inline edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml]: Same as above

**Review by Kevin-OConnor — CHANGES_REQUESTED** [review]
Minor nits, but there is enough confusion with teams about how this all works that I would prefer to be accurate with it.

**Comment by 333fred** [comment]
@frcjenkins ok to test

**Review by Kevin-OConnor — APPROVED** [review]

**Pull request merged**